interface-definitions/include/stunnel/port.xml.i

```diff
@@ -0,0 +1,14 @@
+<!-- include start from stunnel/port.xml.i -->
+<leafNode name="port">
```

Comment:
We already have general https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/port-number.xml.i

Reply:
done